src/Symfony/Component/Messenger/Bridge/Amqp/Transport/Connection.php

```diff
@@ -140,6 +141,9 @@ public function __construct(
      *     * default_publish_routing_key: Routing key to use when publishing, if none is specified on the message
      *     * flags: Exchange flags (Default: AMQP_DURABLE)
      *     * arguments: Extra arguments
+     *     * bindings[name]: An array of the source exchanges to bind this exchange to, keyed by the name. Binding direction: source exchange -> this exchange
+     *       * binding_keys: The binding/routing keys (if any) to be used for the binding
```

Comment:
The keys have to be set (unlike queues) - we could even throw when no keys are defined
```suggestion
     *       * binding_keys: The binding/routing keys to be used for the binding
```